services/controllerhost/extentmon.go

```diff
@@ -524,6 +526,41 @@ func (monitor *extentStateMonitor) handleDestinationExtent(dstDesc *shared.Desti
 			monitor.fixOutOfSyncStoreExtents(dstDesc.GetDestinationUUID(), extent)
 		}
 	}
+	
+	if common.IsRemoteZoneExtent(extent.GetOriginZone(), context.localZone) {
+		monitor.handleRemoteZoneDestinationExtent(dstDesc, isDLQ, extent)
+	}
+}
+
+// handleRemoteZoneDestinationExtent is the local handler for remote destination extent
+// this handler kicks off an event if the primary replicator for the extent is down
+func (monitor *extentStateMonitor) handleRemoteZoneDestinationExtent(dstDesc *shared.DestinationDescription, isDLQ bool, extent *metadata.DestinationExtent) {
+	context := monitor.context
+	// handle remote zone extent replication job failures
+	var failedStores []string
+	for _, storeID := range extent.GetStoreUUIDs() {
+		state, duration := context.failureDetector.GetHostState(common.StoreServiceName, storeID)
+		switch state {
+		case dfddHostStateUnknown:
+			failedStores = append(failedStores, storeID)
+		case dfddHostStateDown:
+			if duration >= maxHostRestartDuration {
+				failedStores = append(failedStores, storeID)
+			}
+		}
+	}
+	if len(failedStores) > 0 {
+		stats, err := context.mm.ReadExtentStats(dstDesc.GetDestinationUUID(), extent.GetExtentUUID())
+		if err != nil {
+			return
```

Comment:
the metadataMgr client already emits a counter for requestCount/failures. So, we will get alerted if metadata calls are failing. And since extentMon is a background job that's run every few minutes, this will be retried anyways.